docs/Homebrew-homebrew-core-Merge-Checklist.md

```diff
@@ -1,22 +1,135 @@
-# Homebrew/homebrew-core Merge Checklist
+# Homebrew/homebrew-core Maintainer Guide
 
-The following checklist is intended to help maintainers decide on
-whether to merge, request changes or close a PR. It also brings more
-transparency for contributors in addition to
-[Acceptable Formulae](Acceptable-Formulae.md) requirements.
+## Quick merge checklist
+
+A detailed checklist can be found [below](#detailed-merge-checklist). This is all that really matters:
+
+- Ensure the name seems reasonable.
+- Add aliases.
+- Ensure it uses `keg_only :provided_by_macos` if it already comes with macOS.
+- Ensure it is not a library that can be installed with
+  [gem](https://en.wikipedia.org/wiki/RubyGems),
+  [cpan](https://en.wikipedia.org/wiki/Cpan) or
+  [pip](https://pip.pypa.io/en/stable/).
+- Ensure that any dependencies are accurate and minimal. We don't need to
+  support every possible optional feature for the software.
+- When bottles aren't required or affected, use the GitHub squash & merge workflow for a single-formula PR or rebase & merge workflow for a multiple-formulae PR. See the ["How to merge without bottles" section below](#how-to-merge-without-bottles)
+for more details.
+- Use `brew pr-publish` or `brew pr-pull` otherwise, which adds messages to auto-close pull requests and pull bottles built by the Brew Test Bot.
+- Thank people for contributing.
+
+Checking dependencies is important, because they will probably stick around
+forever. Nobody really checks if they are necessary or not.
+
+Depend on as little stuff as possible. Disable X11 functionality if possible.
+For example, we build Wireshark, but not the heavy GUI.
+
+Homebrew is about Unix software. Stuff that builds to an `.app` should
+be in Homebrew Cask instead.
+
+## Merging, rebasing, cherry-picking
+
+In most cases, you can simply approve a PR and an automatic merge (with bottles)
+will be performed by [@BrewTestBot](https://github.com/BrewTestBot).
+See [Brew Test Bot For Core Contributors](Brew-Test-Bot-For-Core-Contributors.md) for more information.
+
+PRs modifying formulae that don't need bottles or making changes that don't
+require new bottles to be pulled should use GitHub's squash & merge or rebase & merge workflows.
+See the [table below](#how-to-merge-without-bottles) for more details.
+
+Otherwise, you should use `brew pr-pull` (or `rebase`/`cherry-pick` contributions).
```

Comment:
```suggestion
Otherwise, for example for new formulae, you should use `brew pr-pull` (or `rebase`/`cherry-pick` contributions).
```

Reply:
Is this okay?

```suggestion
Otherwise (e.g. for PRs adding new formula), you should use `brew pr-pull` (or `rebase`/`cherry-pick` contributions).
```

Reply:
Actually, this should probably mention `brew pr-publish` before `brew pr-pull`, right?